network/topics/tracer.go

```diff
@@ -24,15 +24,9 @@ func newTracer(logger *zap.Logger) pubsub.EventTracer {
 
 // Trace handles events, implementation of pubsub.EventTracer
 func (pst *psTracer) Trace(evt *ps_pb.TraceEvent) {
-	pst.report(evt)
 	pst.log(pst.logger, evt)
 }
 
-// report reports metric
-func (pst *psTracer) report(evt *ps_pb.TraceEvent) {
-	metricPubsubTrace.WithLabelValues(evt.GetType().String()).Inc()
-}
-
```

Comment:
just mentioning this was actually pretty useful when we were debugging libp2p pubsub issues. so worth leaving it. I guess its only emitted when we turn on tracer so no harm in leaving?

Reply:
We do have very similar metrics that are also recorded in the P2P Topics domain - `messages in` and `messages out`, although we do not have `event type` as dimension. It was removed because it was high cardinality metric, but I understand it was not enabled by default. There is still a [log written](https://github.com/ssvlabs/ssv/blob/otel-metrics/network/topics/tracer.go#L27), maybe that would be enough for troubleshooting? Currently all the metrics are low cardinality hence we do not have "turn off/on" feature for any of the metrics. I see that as a separate feature which we can include in the next release, so we can come up with the list of metrics that are optional and disabled by default. Wdyt?

Reply:
Let's put it in the backlog so we do not extend the scope of this PR/release if you do not mind